Formula/python@3.11.rb

```diff
@@ -214,7 +215,7 @@ def install
     # `brew install enchant && pip install pyenchant`
     inreplace "./Lib/ctypes/macholib/dyld.py" do |f|
       f.gsub! "DEFAULT_LIBRARY_FALLBACK = [",
-              "DEFAULT_LIBRARY_FALLBACK = [ '#{HOMEBREW_PREFIX}/lib', '#{Formula["openssl@1.1"].opt_lib}',"
```

Comment:
Will need to keep this line (appropriately modified) since we dropped the `openssl@3` commit.